packages/probers/cypress/e2e/uploadTrack.cy.ts

```diff
@@ -202,4 +185,211 @@ describe('Upload', () => {
 
     cy.findByRole('heading', { name: /track/i, level: 1 }).should('exist')
   })
+
+  it('user should be able to upload a single track with stems', () => {
```

Comment:
Could be nice to break out some of the functionality, but this is good for now :)